### PR TITLE
feat: filtering

### PR DESCRIPTION
**Pull request by Kojot999**

### CODE REVIEW

File: src/app/views/Game/Game.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
-export const GameView = () => {
+import styles from "./Game.module.scss";
+import { TailSpin } from "react-loader-spinner";
+
+export const GameView = ({ store }) => {
+  const { data: games, isLoading, error } = store.games;
+  const isContentVisible = !isLoading && !error;
+  console.log(games);
```
Comment (pislagz) [inline src/app/views/Game/Game.js:7]: Do we still need it?

File: src/app/views/Game/Game.js [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
-      </div>
-    </>
+    <div className={styles.wrapper}>
+      {isContentVisible && (
+        <>
+          {/* {games.map((games) => (
+            <div key={games.id}>
+              <h2>a{games.name}</h2>
+            </div>
+          ))} */}
```
Comment (pislagz) [inline src/app/views/Game/Game.js:17]: What is the purpose of this comment?

File: src/app/views/Game/Game.js (hunk 1 above)
Comment (Kojot999) [reply src/app/views/Game/Game.js:7]: its here for development of components

File: src/app/views/Game/Game.js (hunk 2 above)
Comment (Kojot999) [reply src/app/views/Game/Game.js:17]: its still on working progres

File: src/app/components/Sidebar/Dropdown/Dropdown.module.scss [hunk 3]
```diff
@@ -13,10 +13,15 @@
     padding: 0px 10px;
     min-width: 200px;
     justify-content: space-between;
+    h2 {
+      margin: 8px 0px 8px 0px;
```
[the remark this answers is not in the document]
Comment (Kojot999) [reply src/app/components/Sidebar/Dropdown/Dropdown.module.scss]: done

File: src/app/components/Sidebar/Dropdown/Dropdown.js [hunk 4]
```diff
@@ -4,8 +4,14 @@ import { Collapse } from "react-collapse";
...
 import clsx from "clsx";
 
-export const Dropdown = ({ data, name }) => {
+export const Dropdown = ({ data, name, store, queryParam }) => {
   const [isOpened, setIsOpened] = useState(false);
+  const { fetchData: fetchGames } = store.games;
+
+  const handleClick = (id) => {
+    fetchGames({ params: [{ key: queryParam, value: id }] });
+    console.log(id);
```
[the remark this answers is not in the document]
Comment (Kojot999) [reply src/app/components/Sidebar/Dropdown/Dropdown.js]: deleated

File: src/app/api/queryBuilder.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+//   } else  if(searchValue.type==="Platforms"){
+//     const query = "&platforms=" + (searchValue.value);
+//     return RAWG_API + "/games" + API_KEY + query;
+//   } else{}
+//  };
+
+// const exampleParams = [
+//   { key: "page", value: 2 },
+//   { key: "genres", value: "RPG" },
+// ];
```
[the remark this answers is not in the document]
Comment (Kojot999) [reply src/app/api/queryBuilder.js]: deleted

**Pull request merged**